src/traces/bar/plot.js

```diff
@@ -37,8 +37,9 @@ module.exports = function plot(gd, plotinfo, cdbar) {
 
     var bartraces = plotinfo.plot.select('.barlayer')
         .selectAll('g.trace.bars')
-            .data(cdbar)
-      .enter().append('g')
+        .data(cdbar);
+
+    bartraces.enter().append('g')
```

Comment:
@rreusser you probably did this already as part of your animatable bars work, but I couldn't resist making a two-line bug fix to wrap up my day :smirk: